docs/source/usage_guides/multi_node_training.mdx

```diff
@@ -0,0 +1,134 @@
+<!---
+Copyright 2022 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Multi-node Training
+
+Using several Gaudi servers to perform multi-node training can be done easily. This guide shows how to:
+- set up several Gaudi instances
+- set up your computing environment
+- launch a multi-node run
+
+
+## Setting up several Gaudi instances
+
+Two types of configurations are possible:
+- scale-out using Gaudi NICs or Host NICs (on-premises)
+- scale-out using AWS DL1 instances
+
+
+### On premises
+
+To set up your servers on premises, check out the [installation](https://docs.habana.ai/en/latest/Installation_Guide/Bare_Metal_Fresh_OS.html) and [distributed training](https://docs.habana.ai/en/latest/PyTorch/PyTorch_Scaling_Guide/index.html) pages of Habana Gaudi's documentation.
+
+
+### AWS DL1 instances
+
+Follow the following steps to correctly set up your DL1 instances.
+
+
+#### 1. Set up an EFA-enabled security group
+
+To allow all instances to communicate with each other, you need to set up a *security group* as described by AWS in step 1 of [this link](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/efa-start.html#efa-start-security).
+Once this is done, it should look as follows:
+<p align="center">
+    <img alt="Rules of the security group" src="https://huggingface.co/datasets/optimum/documentation-images/resolve/main/habana/multi_node_security_group.png" width="800">
+    <em style="color: grey">Security group for multi-node training on AWS DL1 instances</em>
+</p>
```

Comment:
The alignment issue only appears on Github, but in the doc it will be fine. You can check it here: https://moon-ci-docs.huggingface.co/docs/optimum-habana/pr_116/en/usage_guides/multi_node_training

Reply:
Ah yes for the security group, if I set it the same way as this picture, then I cannot connect to my instances via SSH. So I just followed what is written on AWS and it works: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/efa-start.html#efa-start-security
I think the way you set the security group in your doc works if you launch the run via AWS batch but not if you ssh to the instances as I did.